product_combination_exclude/security/product_attribute_exclude.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Graeme Gellatly
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+<data>
+
+    <record model="ir.model.access" id="product_attribute_exclude_access">
```

Comment:
Any reason that you're not just doing this in CSV? It's much easier and is the standard for the `ir.model.access` rules - https://github.com/OCA/maintainer-tools/blob/master/template/module/security/ir.model.access.csv